examples/memeolist.query.examples.graphql

```diff
@@ -1,10 +1,14 @@
-## MEMES section
 query allMemes {
```

Comment:
This file is no longer used. We could remove it.

Reply:
Can we keep it just for now? I've had some issues with the playground around variables. It tends to be a little buggy sometimes so I'd like to hold onto this file just for a short time

Reply:
Yeah! That was the reason to keep it.
We need variable format as server is now source of truth and we get those queries back to mobile,
but having separate one with variables is also good, but brings more maintenance cost.